### DISCUSSION TITLE
SymbolicUtils versions

### DISCUSSION THREAD

**Discussion by vnikoofard** [MilesCranmer/PySR#307] [Q&A] [answered]
Hi!

Is it possible to change the project.toml for SymbolicUtils = "0.1.0-0.19.7"? The current version in project.toml limits the NeuralPDE.jl

**Comment by MilesCranmer** [comment, marked as answer]
See https://github.com/MilesCranmer/SymbolicRegression.jl/pull/84, https://github.com/MilesCranmer/SymbolicRegression.jl/issues/82, https://github.com/MilesCranmer/SymbolicRegression.jl/issues/59.

**Reply by vnikoofard** [reply]
Thanks! So, for now, we can't have SymbolicRegression and ModelingToolkit in a same environment.

**Reply by MilesCranmer** [reply]
The alternate is to use SymbolicRegression <0.7.0, which allows for newer SymbolicUtils

**Comment by MilesCranmer** [comment]
Fixed on v0.9.0+
